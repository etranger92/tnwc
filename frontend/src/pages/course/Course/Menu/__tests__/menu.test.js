import * as React from 'react';
import MenuFeatureCourse from "../MenuCourse.tsx"
//import { BrowserRouter as Router } from "react-router-dom";
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe("We test the html of Navigation", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(

        <MenuFeatureCourse />

    );
  });
  test("Snapshot menu", () => {
  expect(toJson(wrapper)).toMatchSnapshot();
  });})