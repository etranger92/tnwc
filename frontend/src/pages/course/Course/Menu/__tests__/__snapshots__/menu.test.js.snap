// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`We test the html of Navigation Snapshot menu 1`] = `
<Fragment>
  <ul
    id="menu-list"
  >
    <li
      className=""
      key="menu0"
      onClick={[Function]}
    >
       
      <h3>
        Description
      </h3>
    </li>
    <li
      className=""
      key="menu1"
      onClick={[Function]}
    >
       
      <h3>
        Curriculum
      </h3>
    </li>
    <li
      className=""
      key="menu2"
      onClick={[Function]}
    >
       
      <h3>
        Faq
      </h3>
    </li>
    <li
      className=""
      key="menu3"
      onClick={[Function]}
    >
       
      <h3>
        Announcement
      </h3>
    </li>
  </ul>
</Fragment>
`;
